### ISSUE TITLE
system testing not running for jdk12

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#4851]
Another reason to "fail" a build if no testing runs. I didn't notice this earlier.

Below is one example, but see the entire build at https://ci.eclipse.org/openj9/blue/organizations/jenkins/Pipeline-Build-Test-JDK12/detail/Pipeline-Build-Test-JDK12/5/pipeline/60

https://ci.eclipse.org/openj9/job/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/1
```
07:02:53  Running sanity.system ...
07:02:53  "/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdkbinary/j2sdk-image/bin/java" -version
07:02:53  openjdk version "12-internal" 2019-03-19
07:02:53  OpenJDK Runtime Environment (build 12-internal+0-adhoc.u0020236.Build-JDK12-aixppc-64cmprssptrs)
07:02:53  Eclipse OpenJ9 VM (build master-7d70c4803, JRE 12 AIX ppc64-64-Bit Compressed References 20190223_7 (JIT enabled, AOT enabled)
07:02:53  OpenJ9   - 7d70c4803
07:02:53  OMR      - c3028878
07:02:53  JCL      - af5d06a4dff based on jdk-12+32)
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/daaLoadTest'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/daaLoadTest'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/jlm'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/jlm'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/lambdaLoadTest'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/lambdaLoadTest'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/mathLoadTest'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/mathLoadTest'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/mauveLoadTest'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/mauveLoadTest'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/modularity'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  autoGen.mk:458: warning: overriding recipe for target 'JDKInternalAPIsTest_0'
07:02:53  autoGen.mk:255: warning: ignoring old recipe for target 'JDKInternalAPIsTest_0'
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/modularity'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/otherLoadTest'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/otherLoadTest'
07:02:53  gmake[1]: Entering directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/sharedClasses'
07:02:53  ../systemtest.mk:25: CYGWIN is 0
07:02:53  gmake[1]: Nothing to be done for 'sanity.system'.
07:02:53  gmake[1]: Leaving directory '/home/u0020236/workspace/Test-sanity.system-JDK12-aix_ppc-64_cmprssptrs/openjdk-tests/systemtest/sharedClasses'
07:02:53  
07:02:53  
07:02:53  TEST TARGETS SUMMARY
07:02:53  ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
07:02:53  TOTAL: 0   EXECUTED: 0   PASSED: 0   FAILED: 0   SKIPPED: 0
07:02:53  ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

**Comment by pshipton** [comment]
@smlambert

**Comment by pshipton** [comment]
Closing, assuming fixed.

**Comment by pshipton** [comment]
@smlambert @llxia 
system testing is still not working for jdk12 builds. Possibly we just need #4866 to be ported to the 0.13 branch please.

**Comment by llxia** [comment]
Sorry, I forgot we branched.
@renfeiw Could you please port #4866 into https://github.com/eclipse/openj9/tree/v0.13.0-release? Thanks.

fyi. JDK8 and JDK11 system test builds are fixed.
https://ci.eclipse.org/openj9/view/Test/job/Test-extended.system-JDK11-aix_ppc-64_cmprssptrs/172/tapResults/

**Comment by renfeiw** [comment]
PR to port fix to 0.13.0-release:  #4890

**Comment by pshipton** [comment]
Thanks, it is working now.
